README.md

```diff
@@ -1,2 +1,79 @@
-# action-baseline
-A GitHub Action for running the OWASP ZAP Baseline scan
+# Action Baseline
+
+A GitHub Action for running the OWASP ZAP [Baseline scan](https://github.com/zaproxy/zaproxy/wiki/ZAP-Baseline-Scan). 
+
+The ZAP baseline action scans a target URL for vulnerabilities and maintains an issue in GitHub repository for the
+identified alerts. Read the following ZAP blog for additional information.
+
+
+## Inputs
+
+### `Github token`
+
+**Required** ZAP action uses the default action token provided by GitHub to create and update the issue for the baseline scan.
+
+### `target`
+
+**Required** The URL of the web application to be scanned. This can be either a publicly available web application or a locally
+accessible URL.
+
+### `docker_name`
+
+**Optional** The name of the docker file to be executed. By default the action runs the stable version of ZAP. But you can 
+configure the parameter to use the weekly builds.
+
+### `rules_file_name`
+
+**Optional** You can also specify a relative path to the rules file to ignore any alerts from the ZAP scan. Make sure to create
+the rules file inside the relevant repository. The following shows a sample rules file configuration.
+
+```tsv
+10011	IGNORE	(Cookie Without Secure Flag)
+10015	IGNORE	(Incomplete or No Cache-control and Pragma HTTP Header Set)
+``` 
+
+## Example usage
+
+** Basic **
+```
+steps:
+  - name: ZAP Scan
+    uses: zap/action-baseline@v1
```

Comment:
This would have to be changed to `zaproxy/action-baseline`, right?

Reply:
Changed